### PR TITLE
Prime numbers 

### PR DESCRIPTION
**Pull request by Odigene12**
This should be the correct code for prime numbers. 

@jcockhren 
@ryan-b-writin

### CODE REVIEW

**Comment by ryan-b-writin** [comment]
looks good :+1:

File: Mathmagician.Tests/PrimeNumberTests.cs [hunk 1]
```diff
@@ -19,8 +20,10 @@ public void MakeSureICanMakePrimeNumber()
         [TestMethod]
         public void CheckifPrime()
         {
+            
             PrimeNumber newprime = new PrimeNumber();
 
+            //This will test to see if "2" is a prime number.
             newprime.CheckIfNumberIsPrime(2);
 
             Assert.IsTrue(newprime.CheckIfNumberIsPrime(2));
```
Comment (jcockhren) [inline Mathmagician.Tests/PrimeNumberTests.cs]: @Odigene12  You're calling this method twice. Call `CheckIfNumberIsPrime` once and save the result to variable.

**Comment by jcockhren** [comment]
:+1:

**Comment by BlaiseGratton** [comment]
👍

**Pull request merged**